examples/README.md

```diff
@@ -1,4 +1,68 @@
-# Doorman Integration Examples
+# Integration Examples
 
 - [Python / Flask](python/): A Web UI interacts with Auth0 and a Flask API
 
+# Configuration Examples
+
+## Superusers with Doorman tags
+
+We define a tag ``superuser`` and use it in principals in policies rules.
+
+```yaml
+
+    service: https://api.service.org
+    tags:
+      superuser:
+        - userid:maria
+        - group:admins
+    policies:
+      -
+        id: super-users
+        description: Superusers can do everything
+        principals:
+          - tag:superuser
+        actions:
+          - <.*>
+        resources:
+          - <.*>
+```
+
+## Superusers from service
+
+The superusers are managed on the service itself (eg. database). We will rely on a `superuser` role. The service can send user roles in authorization requests context.
+
+- *TODO: add a small Django demo*
```

Comment:
I'd rather see an example of the kind a curl I would need to run to ask "Is foo@example.com part of the 'superusers' group?"

Reply:
It's not exactly the purpose of Doorman, but we can yes. That would apply to the tag example though.